clients/interactions.py

```diff
@@ -32,8 +33,8 @@ async def send_commands_list(ctx):
     await ctx.send(embed=message)
 
 
-def update_current_squad_player_list(member, before, after):
-    """ Return True if the track question should be sent,
+def send_player_to_squad_player_list(member, before, after):
```

Comment:
Another option would be to prefix `should_`:
`should_add_player_to_players_session_list` (something like this)

`send_player_to_squad_player_list` implies there is an action, but we're only returning a boolean here; no side effects